alfresco-transform-tika/alfresco-transform-tika-boot/src/test/java/org/alfresco/transformer/TikaMetadataExtractsIT.java

```diff
@@ -495,7 +503,6 @@ public void testTransformation(TestFileInfo testFileInfo)
                 //testFile("application/gff", "", ""),
                 //testFile("video/x-oggyuv", "", ""),
                 //testFile("application/x-msdownload", "", ""),
-                testFile(MIMETYPE_IMAGE_JPEG, "jpg", "quick.jpg"),
```

Comment:
Why was "quick.jpg" removed here ?

Reply:
To avoid test duplication as I am already testing jpeg files with more IPTC content above

Reply:
You might as well keep it in. (I know the horse has already bolted here 😄) Maybe add it back in part 2?